app/frontend/src/index.tsx

```diff
@@ -1,33 +1,38 @@
 import React from "react";
 import ReactDOM from "react-dom/client";
-import { HashRouter, Routes, Route } from "react-router-dom";
+import { createHashRouter, RouterProvider } from "react-router-dom";
 import { initializeIcons } from "@fluentui/react";
 
 import "./index.css";
 
 import Layout from "./pages/layout/Layout";
-import NoPage from "./pages/NoPage";
-import OneShot from "./pages/oneshot/OneShot";
 import Chat from "./pages/chat/Chat";
 
 initializeIcons();
 
-export default function App() {
-    return (
-        <HashRouter>
-            <Routes>
-                <Route path="/" element={<Layout />}>
-                    <Route index element={<Chat />} />
-                    <Route path="qa" element={<OneShot />} />
-                    <Route path="*" element={<NoPage />} />
-                </Route>
-            </Routes>
-        </HashRouter>
-    );
-}
+const router = createHashRouter([
```

Comment:
Is it possible to use BrowserRouter instead? (Using the HTML history / pushState APIs). It seems generally preferable over createHashRouter, but maybe that's not compatible with how this app is setup, I haven't tried.

Reply:
Going to leave hash routing, we discussed switching it at the start but decided to keep it.